test/testutil/output.h

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_TU_OUTPUT_H
-# define HEADER_TU_OUTPUT_H
+#ifndef OSSL_TEST_OUTPUT_H
+# define OSSL_TEST_OUTPUT_H
```

Comment:
Nit: `OSSL_TESTUTIL_OUTPUT_H`?  Kinda makes more sense...

The same applies for *all* testutil headers

Reply:
Well, I'm thinking of TESTUTIL as a library (it actually is), so I don't see the need for the added `_TEST_`